README.md

```diff
@@ -1,28 +1,32 @@
 # FizzySteamyMirror
 
-This is a community maintained repo forked from [RayStorm](https://github.com/Chykary/FizzySteamyMirror). Mirror [docs](https://mirror-networking.com/docs/Transports/Fizzy.html).
+This is a community maintained repo forked from **[RayStorm](https://github.com/Chykary/FizzySteamyMirror)**. 
 
-FizzySteamyMirror brings together [Steam](https://store.steampowered.com/) and [Mirror](https://github.com/vis2k/Mirror) utilising Async of a Steam P2P network transport layer for **Mirror**.
+Mirror **[docs](https://mirror-networking.com/docs/Transports/Fizzy.html)** and the official community **[Discord](https://discord.gg/N9QVxbM)**.
+
+FizzySteamyMirror brings together **[Steam](https://store.steampowered.com)** and **[Mirror](https://github.com/vis2k/Mirror)** utilising Async of a Steam P2P network transport layer for **Mirror**.
 
 ## Dependencies
-**Note: SteamWorks.NET is packed with this transport.**
+If you want an easy import, skip the steps bellow & download the latest **[unitypackage](https://github.com/Chykary/FizzySteamyMirror/releases)**, **it has SteamWorks.Net already included**.

```

Comment:
This explains that they should download the **unitypackage**, which then they wont have the issue of wondering why it is not working. The reasoning for this information is because there was people having issues and asking why it's not working, it was because they didn't install **SteamWorks.Net**.